packages/expo-sms/src/SMS.types.ts

```diff
@@ -0,0 +1,3 @@
+export type SMSResponse = {
+  result: 'unknown' | 'sent' | 'cancelled';
```

Comment:
IMO it would be nice to provide some `ENUM` holding these possible `result` values.
e.g.
```typescript
export enum SMSResult {
  Unknown = 'unknown',
  Sent = 'sent',
  Cancelled = 'cancelled'
}

export type SMSResponse = {
  result: SMSResult;
}
```

Ideally it would be wonderful if TS would provide some mechanism for generating UnionType consisting of `SMSResult` itself and it's string values (`'unknown' | 'sent' | 'cancelled'`) by some similar construction as `keyof typeof SMSResult` (resulting in `"Unknown" | "Sent" | "Cancelled"`), but for values instead of keys.

Above would allow users to compare results both by Enum (`smsResponse.result === SMSResult.Sent`) and string (`smsResponse.result === 'sent'`).

Of course we can add these strings manually to desired UnionType (`result: SMSResult | 'unknown' | 'sent' | 'cancelled'`) and probably that the best solution 😛

What do you think?

Maybe it's just overengineering the problem as there seems to be other ways to ensure exhaustive check for union types consisted of strings 😛 

cc @ide

Reply:
We could do an enum like that for now since it doesn't change the runtime API. This API hasn't been part of the SDK audit yet so we don't want to make any big changes.

Reply:
This isn't an audit. Just moving the types to a new file.